server/src/router/index.js

```diff
@@ -6,12 +6,21 @@ const {
   getCoursesByCatId,
   getCourseDetails,
   getFavorite,
+  googleLogin,
+  verifyUser,
 } = require('../controllers');
 
 router.get('/:categoryId/courses', getCoursesByCatId);
 router.get('/courses/:courseId', getCourseDetails);
 router.get('/:categoryId/courses', getCoursesByCatId);
 router.get('/favorite/:userId', getFavorite);
+router.post('/login/google', googleLogin);
+router.use(verifyUser);
```

Comment:
favorite route will be protected so, fix this : 
```suggestion
router.post('/login/google', googleLogin);
router.use('/favorite/:userId', verifyUser);
router.get('/favorite/:userId', getFavorite);
```